DashboardClient/DashboardClient.cpp

```diff
@@ -26,6 +26,213 @@ namespace Umati
 			std::shared_ptr<OpcUaTypeReader> pTypeReader)
 			: m_pDashboardDataClient(pDashboardDataClient), m_pPublisher(pPublisher), m_pTypeReader(pTypeReader)
 		{
+			this->startEventThread();
+		}
+
+		DashboardClient::~DashboardClient() {
```

Comment:
Best practice is following the rule of tree.

https://en.cppreference.com/w/cpp/language/rule_of_three